### ISSUE TITLE
SwitchButton setEnabled(false) makes onTouchEvent to return false

### ISSUE THREAD

**Issue by igormorse** [kyleduo/SwitchButton#98]
Hello, I was working with this library for a couple mounths and just now I realized that if you run the method **setEnabled(false)** it prevents you from clicking in the Switch passing the clicking event to the parent.

Here is a code sample in **SwitchButton.java**: 

```
@Override
public boolean onTouchEvent(MotionEvent event) {

    if (!isEnabled() || !isClickable() || !isFocusable()) {
        return false;
    }
```

If you check the documentation for `onTouchEvent()` , you'll find:

> Returns: True if the event was handled, false otherwise.

Is this the expected behaviour ? I would like to block the Switching thing but I also wanted to be able to perform clicking on it but not doing the OnChecked thing.

**Comment by kyleduo** [comment]
Yes, and there is no standalone configuration to disable switch gesture. 
I think switch gesture is a good interaction and no need to disable it. 
However, the latest version, 2.0.0, change this behavior a little and if you use it in a scrollable view, SwitchButton will ignore vertical gestures so scrollable view will works fine. If this is the reason of you want to disable the gesture, you can update to the latest version and see if it has solved your problem.
